Ambiplatforms-TORQUE/ethermint
# tests(rpc): add filter tests

## Description

* tests(rpc): add pending transaction filter test

* tests(rpc): add block filter and event log  test

* tests(rpc): simplify to cluster instead of comparing types

* tests(rpc): wip filter by address

* tests(rpc): add get_logs test

* fix flake8 linter

* fix flake8 linter

* add caching to readme

* add caching to readme
______

For contributor use:


______

For admin use:
